tools/README.md

```diff
@@ -0,0 +1,38 @@
+# Tools
+
+This directory contains a set of tools to facilitate certain tasks that a Che developer may encounter.
+
+## Plugin Updates
+
+All Che plugins depend on VS Code extensions, and these are usually hosted on other repositories. Sometimes these extensions can be out of date,
+as a newer version of the extension has been released upstream. In order to maintain the Che Plugin Registry and keep it up to date, a script
+has been provided which can check for updates, report which plugins need updating, and automatically file issues for those plugins that do
+need updating.
+
+Installation: create a virtual environment with the following Python packages installed
```

Comment:
Should you provide Dockerfile with the required runtime for this script ?

Reply:
Sure, that's an interesting idea!

Reply:
@benoitf just to clarify, what is your intended use case? Someone would be able to build the container and run the script from in there? Should the token/repository details be editable in the Dockerfile, or as environment variables when running the script?

Reply:
use case was to run the script packaged into a container and by just providing env variable (so the container can run everywhere and doesn't contain any token/secrets) I'm able to use it